sunpy/map/mapbase.py

```diff
@@ -679,13 +679,21 @@ def _base_name(self):
     @property
     def name(self):
         """Human-readable description of the Map."""
-        return self._base_name().format(measurement=self.measurement)
+        if self.measurement == 0 * u.Angstrom:
```

Comment:
My real question here is not that this isn't an improvement (it definitely is) but why are we defaulting the measurement to `0 * u.AA` rather than `None`?